components/form/CheckboxGroup.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { useFormContext } from 'react-hook-form';
+import PropTypes from 'prop-types';
+import { CheckboxLabel, ErrorLabel } from '.';
+
+export function Checkbox({ ...props }) {
+  const { register } = useFormContext();
+  return (
+    <>
+      <div className={`flex items-center ${props.className}`}>
+        <input
+          id={`checkbox-${props.value}`}
+          className="w-4 h-4 text-amcovad-primary-400 border-gray-200 rounded focus:ring-amcovad-primary-300 "
+          name={props.name}
```

Comment:
We should use name directly:

```suggestion
          name={name}
```

This should also be fixed in other places `props.name` is used.